Jenkinsfile

```diff
@@ -209,7 +209,10 @@ pipeline {
 									withEnv(["JAVA_HOME=${WORKSPACE}/jdk.resources", "EXE_OUTPUT_DIR=${WORKSPACE}/libs", "LIB_OUTPUT_DIR=${WORKSPACE}/libs"]) {
 										dir(ws) {
 											if (isUnix()) {
-												sh "sh build.sh -ws ${ws} -os ${os} -arch ${arch} checklibs install"
+												sh """
+												sh build.sh clean
+												sh build.sh -ws ${ws} -os ${os} -arch ${arch} checklibs install
+												"""
```

Comment:
Why is this change necessary?
The containers should be empty at the beginning of each run.